### DISCUSSION TITLE
GlobalFilter and hide columns together

### DISCUSSION THREAD

**Discussion by JustynaKK** [TanStack/table#2450] [General]
I wanted to implement the globalFIlter and hiding columns together.
When I hide some columns, I expected global filters not search through their values, however, globalFilters stilll search through all the columns, regardless if hidden or not.

Is there the way to pass to globalFIlter parameter to search only through visible columns?

**Comment by tannerlinsley** [comment]
I converted this over to a discussion. :) Also, feel free to use the discord server to ask questions! https://discord.com/invite/WrRKjPJ
